### ISSUE TITLE
XIAS process deprecated on 2023-10-01

### ISSUE THREAD

**Issue by wwarriner** [uabrc/uabrc.github.io#609]
### What would you like to see added?

The XIAS account process is being deprecated as of 2023-10-01.

Existing XIAS accounts are valid until their expiration date, but new ones may not be created and expiration may not be extended. There is a new process we'll need to learn and link to/document.

For our purposes, all XIAS researchers needing continued access to our system will need to have employee category 5 (affiliate) or 60 (volunteer). This can be managed through ACT documents by the responsible party (i.e., the PI).

This link has more info for various ACT documents: https://www.uab.edu/humanresources/home/technology-data-services/oracle-administrative-systems-training/oracle-uab-hr-officer/act-document-types-reasons.

The bottom line is that XIAS users will need to have PIs "hire" them through departmental or central HR, recorded as affiliates or volunteers. We can't control this process and it is outside the scope of our responsibility. Once they are affiiliates or volunteers, they are free to get access to our system in the usual way.

**Comment by wwarriner** [comment]
Hold off on this one, this entire thing is in flux and more information is coming.
